ui/imports/shared/popups/UserAgreementPopup.qml

```diff
@@ -0,0 +1,121 @@
+    hasCloseButton: false
+
+    closePolicy: Popup.NoAutoClose
+
+    component Paragraph: StatusBaseText {
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        font.pixelSize: 15
+        lineHeightMode: Text.FixedHeight
+        lineHeight: 22
+        visible: !!text
+        wrapMode: Text.Wrap
+    }
+
+    component AgreementSection: ColumnLayout {
+        property alias title: titleItem.text
+        property alias body: bodyItem.text
+        spacing: 12
+        Paragraph {
+            id: titleItem
+            font.weight: Font.DemiBold
+        }
+
+        Paragraph {
+            id: bodyItem
+        }
+    }
+
+    StatusScrollView {
+        id: scrollView
+        anchors.fill: parent
+        contentWidth: availableWidth
+
+        ColumnLayout {
+            id: layout
+            width: scrollView.availableWidth
+            spacing: 22
+            StatusRoundedImage {
+                id: statusRoundedImage
+                objectName: "headerImage"
+                Layout.preferredWidth: 64
+                Layout.preferredHeight: 64
+                Layout.alignment: Qt.AlignHCenter
+                image.source: Style.png("status-logo")
+            }
+
+            AgreementSection {
+                title: qsTr("Warning - Status desktop is currently in Beta")
+                body: qsTr("•  The Wallet is not yet safe or secure to use
+•  Do not use real funds in the Wallet
+•  Do not use accounts that contain tokens of value in the Wallet")
+            }
+
+            AgreementSection {
+                title: qsTr("Also be aware")
+                body: qsTr("•  Status Desktop is incompatible with Status Mobile versions 1.x
+•  This version of Status may break or stop working without warning
+•  Communities created with this version may be broken by future releases
+•  Status desktop currently consumes large amounts of bandwidth")
+            }
+
+            AgreementSection {
+                body: qsTr("We are working to fix all these issues ASAP, ahead of Status Desktop’s 1.0 release!")
+            }
+
+            StatusDialogDivider {
+                Layout.fillWidth: true
+            }
+
+            AgreementSection {
+                body: qsTr("I confirm that...")
+            }
+
+            StatusCheckBox {
+                id: agreeToUse
+                Layout.fillWidth: true
+                Layout.topMargin: -10 //reduced margin by design
+                contentItem: Paragraph {
+                    text: qsTr("I’ve read the above and understand that Status Desktop is Beta software")
+                    leftPadding: readyToUse.indicator.width + readyToUse.spacing
+                }
+            }
+
+            StatusCheckBox {
+                id: readyToUse
+                Layout.fillWidth: true
+                Layout.topMargin: -10 //reduced margin by design
+                Layout.bottomMargin: layout.spacing
+                contentItem: Paragraph {
```

Comment:
maybe
```
        Layout.fillWidth: true
        Layout.fillHeight: true
```
could be moved from `Paragraph` to `AgreementSection` bc in this context those attached properties make no sense.